application/client.core/src/app/environment/components/sidebar/shell/running/component.ts

```diff
@@ -0,0 +1,103 @@
+import { NotificationsService } from '../../../../services.injectable/injectable.service.notifications';
+import { IShellProcess} from '../../../../../../../../common/ipc/electron.ipc.messages';
+import { Component, OnDestroy, ChangeDetectorRef, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
+
+import ElectronIpcService, { IPCMessages } from '../../../../services/service.electron.ipc';
+import ContextMenuService, { IMenuItem } from '../../../../services/standalone/service.contextmenu';
+import ShellService from '../services/service';
```

Comment:
It's a great idea/way to organize communication and create an own service for shell. I have two suggestions, just to complete this idea:
- as soon as this service is related only to shell, let's put it into a component folder (see `charts` as an example)
- service can care about all IPC calls and inside components you can call it as simple promises. The benefit is: you will have all IPC calls in one place. It would easy to debug and modify. 
-